src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -21,13 +22,15 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the recipe identified by the name used in the displayed list of recipes.\n"
             + "Parameters: name (must be a valid case sensitive name)\n"
-            + "Example: " + COMMAND_WORD + " Aglio Olio";
+            + "Example: " + COMMAND_WORD + " Aglio Olio\n"
+            + "Example: " + COMMAND_WORD + " -i 3";
```

Comment:
I don't think i have an issue with the usage of the tag "-i" here to specify index, just wondering if it's going to be a problem if "-i" is being used for ingredients as well.

Reply:
@blaketingyu I was considering that as well, but as the user, "-i" would only trigger the Ingredient Prefix if the command is an `AddCommand`.

Reply:
Yeap right,  i was thinking of having no prefix but at the same time, I don't mind having "-i" used for index as well.  Also, "-i" will trigger the ingredient prefix if the command is an `EditCommand` that we are implementing soon.